freud/CMakeLists.txt

```
@@ -86,8 +86,6 @@ foreach(cython_module ${cython_modules_with_cpp} ${cython_modules_without_cpp})
   else()
     set_target_properties(${cython_module} PROPERTIES INSTALL_RPATH "\$ORIGIN")
   endif()
```

Revert this if Mac builds fail.
```suggestion
  endif()
  set_target_properties(${cython_module} PROPERTIES INSTALL_RPATH_USE_LINK_PATH
                                                    True)
```